### PR TITLE
Request Pull request and Code Review

### PR DESCRIPTION
**Pull request by konyan**

### CODE REVIEW

**Comment by shshamim63** [comment]
# Status :arrow_forward: [Changes Required] :arrow_backward: 
___
Hi @konyan :wave: ,
Great job :clap: , I can see all the parts that are needed to decorate the page are in here. Awesome initiative :open_hands: . But there are some issues that are blocking me to declare this project as complete. 
#### Please Follow the instructions below :point_down: to complete the project
___
:warning:  As far I know this project is collaborative project. But based on the commit history i can see only you are alone working on this project. Do you have the permission to work alone on this project? Without  the permission your project will be considered as invalid. 
___
## [Mandatory Changes]
___
### [Large version]
#### !Top 3 articles section!
1. Well on my screen i can see that it is not decorated as the original webpage. Cause a class has been added when the width is more than 1279px. Take a look at this :point_down: picture. 
picture1
2. The latest news section should contain 4 items in a row. But i can see 3. That is because you have declared that each item should contain exact 320px , so that is predefined. Which means what the other end is one should contain 320px width. Well i will suggest when ever you want to define width it is better to use percentage(%) cause it adjust itself based on the end device.
picture2
3. Latest findings section, also same as above :point_up: .
4. The footer icons are really compact and the links are containing more space. Which look awkward. I think you should improve that thing too.
picture3
___

### [Medium version]
If I am not wrong the medium version starts from 768px, Check this [link](https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints/). So you have play with your CSS that are handling the medium screen. I am sure you can fix this issue.

 ### [Small version]
1. The Latest funding portion is broken. I can feel that it is saying `I need some attention` :smile: 
picture4
___

#### Kindly fix the mentioned issues and send another review request
___
### :clubs: Happy Coding :clubs: 
#### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

**Comment by konyan** [comment]
> # Status ▶️ [Changes Required] ◀️
> Hi @konyan 👋 ,
> Great job 👏 , I can see all the parts that are needed to decorate the page are in here. Awesome initiative 👐 . But there are some issues that are blocking me to declare this project as complete.
> 
> #### Please Follow the instructions below 👇 to complete the project
> ⚠️ As far I know this project is collaborative project. But based on the commit history i can see only you are alone working on this project. Do you have the permission to work alone on this project? Without the permission your project will be considered as invalid.
> 
> ## [Mandatory Changes]
> ### [Large version]
> #### !Top 3 articles section!
> 1. Well on my screen i can see that it is not decorated as the original webpage. Cause a class has been added when the width is more than 1279px. Take a look at this 👇 picture.
>    picture1
> 2. The latest news section should contain 4 items in a row. But i can see 3. That is because you have declared that each item should contain exact 320px , so that is predefined. Which means what the other end is one should contain 320px width. Well i will suggest when ever you want to define width it is better to use percentage(%) cause it adjust itself based on the end device.
>    picture2
> 3. Latest findings section, also same as above ☝️ .
> 4. The footer icons are really compact and the links are containing more space. Which look awkward. I think you should improve that thing too.
>    picture3
> 
> ### [Medium version]
> If I am not wrong the medium version starts from 768px, Check this [link](https://responsivedesign.is/develop/browser-feature-support/media-queries-for-common-device-breakpoints/). So you have play with your CSS that are handling the medium screen. I am sure you can fix this issue.
> 
> ### [Small version]
> 1. The Latest funding portion is broken. I can feel that it is saying `I need some attention` 😄
>    picture4
> 
> #### Kindly fix the mentioned issues and send another review request
> ### ♣️ Happy Coding ♣️
> #### Shakhawat Hossain
> [Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

thanks for your feedback. i already fix and update its. Please check

**Comment by Anna-Myzukina** [comment]
#### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi @konyan!  
#### Wow, your project looks :+1: 
#### But need some Required Changes

### Page structure
#### Navigation bar
##### Top 3 articles section

* [x] It contains 3 articles with an image as a background.

###### Large version

* [ ] It contains one bigger image on the leftmost part of the row (around two thirds of the width).
* [x] It contains two smaller images on the rightmost part of the row (one above another).
* [x] It contains TWN logo at the top left corner of the row.

*** As you can see in original on the leftmost part of the row one picture takes around two thirds of the width ***

tnw

##### Latest news section

###### Large version

* [x] It contains a header with the section name.
* [ ] It aligns 4 articles per row.
* [ ] Each article has a cover image and title placed below the image.

###### Medium version changes

* [ ] It aligns 3 articles per row.

###### Small version changes

* [x] It aligns 1 articles per row.
* [x] Each articles has a cover image on the left side and title on the right side of the row.

##### Latest Funding section

###### Large version

* [x] It contains a header with the section name.
* [ ] It aligns 4 articles per row.

###### Medium version changes

* [ ] It aligns 3 articles per row.

###### Small version changes

* [x] It aligns 1 articles per row.

##### Latest deals section

###### Large version

* [x] It contains a header with the section name.
* [ ] It aligns 4 deals per row.

###### Medium version changes

* [ ] It aligns 3 deals per row.

###### Small version changes

* [x] It aligns 1 deal per row.

Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding 😄*

File: css/news-three.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+@media (max-width: 767px) {
+  .news--three--list {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(0px, 1fr));
+    grid-gap: 1rem;
+    margin-bottom: 3rem;
+  }
+} */
```
Comment (Redvanisation) [inline css/news-three.css]: Please delete commented code

Comment (konyan) [reply css/news-three.css]: DONE

**Comment by Anna-Myzukina** [comment]
### Your project steel needs changes in Page structure

#### Navigation bar
##### Top 3 articles section

###### Large version

* [ ] It contains one bigger image on the leftmost part of the row (around two thirds of the width).

*** As you can see in original on the leftmost part of the row one picture takes around two thirds of the width ***

tnw

*** In your project it`s half of screen ***

half


##### Latest news section

###### Large version

* [ ] It aligns 4 articles per row.

*** In your project here just 3 articles per row ***

latest



##### Latest Funding section

###### Large version

* [ ] It aligns 4 articles per row.

*** Here should be 4 articles, but your project has 3 ***

funding


###### Medium version changes

* [ ] It aligns 3 articles per row.

*** In medium version your project has 2 articles per row, but need to be 3 ***
medium

##### Latest deals section

###### Large version

* [ ] It aligns 4 deals per row.
*** Large version on your project has just 3, but need to be 4

deel



### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding 😄*

**Comment by Anna-Myzukina** [comment]
### Few sections have previous errors in Page structure.

#### Navigation bar

##### Latest news section

###### Large version

* [ ] It aligns 4 articles per row.

*** In your project here just 3 articles per row, but should be 4 ***

latest



##### Latest Funding section

###### Large version

* [ ] It aligns 4 articles per row.

*** Here should be 4 articles, but your project has 3 ***

funding


###### Medium version changes

* [ ] It aligns 3 articles per row.

*** In medium version your project has 2 articles per row, but need to be 3 ***
medium

##### Latest deals section

###### Large version

* [ ] It aligns 4 deals per row.
*** Large version on your project has just 3, but need to be 4

deel



### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding 😄*

**Comment by konyan** [comment]
@Anna-Myzukina can u recheck about project , I already remove Latest Funding section one item, currently it is only 3 item in Latest Funding section.

**Comment by konyan** [comment]
image

**Pull request merged**